ntoskrnl/ke/amd64/except.c

```diff
@@ -147,7 +147,7 @@ KiDispatchExceptionToUser(
 
         /* Nothing we can do here */
         _disable();
-        _SEH2_YIELD(return);
+        return FALSE;
```

Comment:
So, do we still need SEH2_YIELD or not anymore, when we return from within SEH blocks?

Reply:
No, it's no longer needed.

Reply:
Yes my concern here was about non-MSVC compilers (GCC with PSEH, basically).